plugin/trino-pinot/src/test/java/io/trino/plugin/pinot/TestPinotConnectorSmokeTest.java

```diff
@@ -2799,4 +2799,11 @@ public void testQueryOptions()
                 .matches("VALUES (VARCHAR 'Los Angeles', DOUBLE '50000.0'), (VARCHAR 'New York', DOUBLE '20000.0')")
                 .isFullyPushedDown();
     }
+
+    @Test
+    public void testTableFunctions()
```

Comment:
Could you add more test cases? You can refer to `testNative*` methods in `BaseJdbcConnectorTest` class. 

I see `select 1 as a from region` works in Pinot query console, but this table function throws an exception.